<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWhirrUpdaterMainWindow</class>
 <widget class="QMainWindow" name="AppWhirrUpdaterMainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AppWhirrU</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	background-color: #FFFFFF;

border-style: solid;
	border-width: 1px;
	border-color: #737373;
	border-radius: 2px;
}</string>
   </property>
   <widget class="QWidget" name="progressBackground" native="true">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>448</width>
      <height>38</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background: url(:/progressBarBackground);
}</string>
    </property>
    <widget class="QWidget" name="progressBar" native="true">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>444</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background: url(:/progressBarIndicator);
	border-radius: 2px;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>169</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* { 
color: #666666;
font: 14px arial, sans-serif;
}</string>
    </property>
    <property name="text">
     <string>AppWhirr is being updated,
Please be patient.</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>128</y>
      <width>451</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* { 
color: #9d9d9d; 
font: 14px arial, sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="progessLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>448</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background: transparent;
	font: 20px sans-serif;
	color: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="lightLogo" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>1</y>
      <width>245</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-image: url(:/lightLogo);
}</string>
    </property>
   </widget>
   <zorder>lightLogo</zorder>
   <zorder>progressBackground</zorder>
   <zorder>label</zorder>
   <zorder>statusLabel</zorder>
   <zorder>progessLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
